[core]/esx_multicharacter/server/main.lua

```diff
@@ -36,8 +36,14 @@ local function GetIdentifier(source)
         return "ESX-DEBUG-LICENCE"
     end
     
-    local identifier = string.gsub(GetPlayerIdentifierByType(source, PRIMARY_IDENTIFIER), PRIMARY_IDENTIFIER .. ':', '')  
-    return identifier 
+    local identifier = GetPlayerIdentifierByType(source, PRIMARY_IDENTIFIER)
+
+    if not identifier then
+        return
+    end 
+    
+    identifier = identifier:gsub(PRIMARY_IDENTIFIER .. ':', '')
+    return identifier
```

Comment:
return identifier:gsub(PRIMARY_IDENTIFIER .. ':', '') enough, don't need pass to variable, you don't use identifier afterwards.

Reply:
i did that to avoid the extra number behind the identifier that would be returned

Reply:
> i did that to avoid the extra number behind the identifier that would be returned

It's okey, i forget gsub return 2 values.